addons/forum-post-snip/addon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Collapse nested forum quotes",
+  "description": "Collapses forum quotes",
```

Comment:
Too short and does not describe the functionality well.

Reply:
that's todo